src/game-service/entities/player.js

```diff
@@ -1992,10 +1992,104 @@ Player.onConnect = function(socket, cognitoSub, name, team, partyId){
 	});
 }//End Player.onConnect
 
+const PERM_NONE = 0;
+const PERM_OWNER = 1;
+const PERM_MODERATOR = 2;
+const PERM_DEVELOPER = 3;
+function getPerm(cognitoSub) {
+	let ret = PERM_NONE;
+	if (cognitoSub == createdByCognitoSub)
```

Comment:
createdByCognitoSub is undefined is it not?

Reply:
I could add a global variable that gets set when custom server is created (and reset to "" when the server goes back to ranked). All the custom server settings are global variables in config.js, and the createdByCognitoSub could be one of them.

Reply:
By the way, cognitoSub is an Amazon Cognito attribute. Basically a unique key tied to your authenticated user.